src/email_templates/report_approved/html.pug

```diff
@@ -1,8 +1,8 @@
 p Hello,
-p 
-  | Activity Report #{displayId} has been approved by #{manager}.
```

Comment:
I know these aren't going out yet (so just thinking out loud) but I could see users wanting to know which manager approved their report, and maybe which managers still need to approve.

Reply:
I think that's a good idea. Let's leave it for a future iteration though. I have a feeling as soon as notifications are live we're going to have a list of tweaks and changes.